modules/javascript/partytown-web-worker/can-load.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * Can load function to determine if Partytown Web Worker module can be loaded.
+ *
+ * @since   n.e.x.t
+ * @package partytown-web-worker
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+if ( ! function_exists( 'perflab_partytown_web_worker_can_load' ) ) {
+
+	/**
+	 * Check if Partytown Web Worker module can be loaded.
+	 *
+	 * @since n.e.x.t
+	 * @return bool|WP_Error True if the module can be loaded, WP_Error otherwise.
+	 */
+	function perflab_partytown_web_worker_can_load() {
+		$errors = new WP_Error();
+
+		// Check for Partytown JavaScript library.
+		if ( ! file_exists( __DIR__ . '/assets/js/partytown/partytown.js' ) ) {
+			$errors->add(
+				'partytown_web_worker_library_missing',
+				sprintf(
+					/* translators: %s: npm install && npm run build-plugins */
+					__( 'Partytown library is missing from the plugin. Please do %s to install it.', 'performance-lab' ),
+					wp_kses(
+						'<code>npm install &amp;&amp; npm run build-plugins</code>',
+						array(
+							'code' => array(),
+						)
+					)
+				)
+			);
+		}
+
+		// Check for HTTPS.
+		if (
+			! is_ssl()
+			&&
+			( strpos( get_bloginfo( 'wpurl' ), 'https' ) !== 0 )
```

Comment:
Passing `wpurl` is just an alias for `site_url()`:

```suggestion
			( wp_parse_url( site_url(), PHP_URL_SCHEME ) !== 'https' )
```